Replace Jakarta activation with angus activation

There seems to be a problem with Jakarta activation, which causes an error upon deployment: 

`Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [de.imi.mopat.helper.controller.MailSender]: Factory method 'mailSender' threw exception with message: Provider for jakarta.activation.spi.MimeTypeRegistryProvider cannot be found
`

Replacing the dependency with angus activation seems to fix the issue: 

```
<dependency>
    <groupId>org.eclipse.angus</groupId>
    <artifactId>angus-activation</artifactId>
    <version>2.0.2</version>
</dependency>
```
https://mvnrepository.com/artifact/org.eclipse.angus/angus-activation

--- a/pom.xml
+++ b/pom.xml
@@ -291,9 +291,9 @@
             <version>1.12.5</version>
         </dependency>
         <dependency>
-            <groupId>jakarta.activation</groupId>
-            <artifactId>jakarta.activation-api</artifactId>
-            <version>2.1.2</version>
+            <groupId>org.eclipse.angus</groupId>
+            <artifactId>angus-activation</artifactId>
+            <version>2.0.2</version>
         </dependency>
         <dependency>
             <groupId>jakarta.el</groupId>

